src/main/java/com/gruzini/tennistico/mappers/MatchInfoParser.java

```diff
@@ -0,0 +1,70 @@
+package com.gruzini.tennistico.mappers;
+
+import com.gruzini.tennistico.domain.Match;
+import com.gruzini.tennistico.domain.Player;
+import com.gruzini.tennistico.domain.Score;
+import com.gruzini.tennistico.exceptions.PlayerNotFoundException;
+import com.gruzini.tennistico.services.entity_related.MatchService;
+import com.gruzini.tennistico.services.entity_related.ScoreService;
+import org.springframework.stereotype.Component;
+
+import static java.util.Objects.isNull;
+
+@Component
+public class MatchInfoParser {
+    private final MatchService matchService;
+    private final ScoreService scoreService;
+
+    public MatchInfoParser(MatchService matchService, ScoreService scoreService) {
+        this.matchService = matchService;
+        this.scoreService = scoreService;
+    }
+
+    public String getScore(final Long matchId, final Player player) {
```

Comment:
Rozumiem, że sprawdzenie, czy match istnieje jest stricte pod notyfikacje używane, tak?

Reply:
no a ma wywalić exception? Taki przypadek że meczu nie ma a leci notyfikacja dla niego jest mało prawdopodobny.

Reply:
Pytałem, bo ja moje funkcjonalności też z tego korzystają i się zastanawiałem, czy u mnie też mógłby być przypadek, że mecz by nie istniał. Raczej niemożliwe, ale mogłem pominąć jakiś przypadek.

Reply:
jest to double check ale dzięki temu możemy zawsze z tego skorzystać mając jedynie id meczu